src/native/corehost/apphost/apphost.windows.cpp

```diff
@@ -81,6 +82,7 @@ namespace
         {
             // We don't have a great way of passing out different kinds of detailed error info across components, so
             // just match the expected error string. See fx_resolver.messages.cpp.
+            dialogMsg = pal::string_t(_X("To run this application, you must install .NET.\n\n"));
```

Comment:
I would probably reword this to something like: "To run this application, you must install missing framework for .NET." - The "missing .NET" case is handled above, this is the "missing framework case" (and we MUST NOT use the ".NET framework" wording - since that would make it sound like the .NET Framework product.